framework/interface/templates/plugin_report.html

```diff
@@ -94,7 +95,7 @@ <h4 id="header_badge_{{ test_code }}"></h4>
                                         <td> {{ poutput['start_time'] }} <br />{{ poutput['end_time'] }} </td>
                                         <td> {{ poutput['status'] }} </td>
                                         {% if poutput['output_path'] %}
-                                            <td><a href="/output_files/{{ poutput['output_path'] }}/" target="_blank" class="btn btn-primary">Browse</a></td>
+                                            <td><a href="/output_files//{{ quote_plus(poutput['output_path'], safe='/') }}/" target="_blank" class="btn btn-primary">Browse</a></td>
```

Comment:
By intercepting the request in burp suite I observed that if I dont put /(or any other character) then it is not making the correct GET request but I am not able reproduce it now :confused: .You check that @delta24  and if everything is fine make the appropriate changes.